service/src/main/java/com/codeforcommunity/requester/MapRequester.java

```diff
@@ -242,6 +244,16 @@ public void updateBlocks(List<String> blockIds, BlockStatus updateTo) {
         });
   }
 
+  public Object getPrivateStreets() {
+    try {
+      return new JSONParser()
+          .parse(new FileReader("service/src/main/resources/private_streets_light.geojson"));
+    } catch (Exception e) {
+      logger.error("There was an error loading GeoJSON data", e);
```

Comment:
Let's make this error message more specific, noting that it's private street data